src/StackAdmin/Azs.Storage.Admin/Module/Azs.Storage.Admin/Get-TaskResult.ps1

```diff
@@ -7,7 +7,7 @@ Microsoft.PowerShell.Core\Set-StrictMode -Version Latest
 
 <#
```

Comment:
There should not be a need to touch this file. How did you create the powershell? Did you hand edit or ran the PsSwagger tool?

Reply:
I generate it via generate.ps1 under the folder. It seems calling the psswagger to do so. Not sure why it is changed, I'll revert the change.